modules/metrics.py

```diff
@@ -1,15 +1,48 @@
 import enum
-
 import prometheus_client
 
 
+class ExitStatus(enum.Enum):
+    SUCCESS = (0, "success")
+    GENERIC_ERROR = (1, "generic_error")
+    INVALID_ARGUMENT = (2, "invalid_argument")
+    OUT_OF_MEMORY = (137, "out_of_memory")
+    SEGMENTATION_FAULT = (139, "segmentation_fault")
+    KILL = (-9, "kill")
+    UNKNOWN_STATUS = (None, "unknown_error")
+
+    def __init__(self, code, label):
+        self.code = code
+        self.label = label
+
+    @staticmethod
+    def from_exit_code(code):
+        for status in ExitStatus:
+            if status.code == code:
+                return status
+        return ExitStatus.UNKNOWN_STATUS
+
```

Comment:
can we remove this code

Reply:
then we are using exit code (as integers) for their labels right?